src/index.html

```diff
@@ -7,8 +7,44 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet"
+    href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="navigation">
+      <a href="index.html" class="nav-logo">
+        <img src="images/logo.png" alt="moyo_logo"></img>
+      </a>
+      <nav class="navigation-box">
+        <ul class="nav__list">
+          <li>
+            <a href="#apple" class="nav__link nav__link--active is-active">apple</a>
+          </li>
+          <li>
+            <a href="#samsung"
+            class="nav__link">samsung</a>
+          </li>
+          <li>
+            <a href="#smartphones" class="nav__link">smartphones</a>
+          </li>
+          <li>
+            <a data-qa="hover"
+            href="#laptops&computers" class="nav__link">laptops & computers</a>
+          </li>
+          <li>
+            <a href="#gadgets" class="nav__link">gadgets</a>
+          </li>
+          <li>
+            <a href="#tablets" class="nav__link">tablets</a>
+          </li>
+          <li>
+            <a href="#photo" class="nav__link">photo</a>
+          </li>
+          <li>
+            <a href="#video" class="nav__link nav__link--last">video</a>
```

Comment:
I can't really see the usege of that second class - please remove it, if not necessary 😉